src/component_managers/astt_comp_manager.py

```diff
@@ -64,24 +64,32 @@ def az_el_change_callback(self, incoming_object):
                 node_record.name
                 == "Position Feedback.Azimuth(R64) of position"
             ):
-                (self.logger).info(f"AZ = {node_record.raw}")
+                pass
 
             if (
                 node_record.name
                 == "Position Feedback.Elevation(R64) of position"
             ):
-                (self.logger).info(f"EL = {node_record.raw}")
+                pass
 
     # This is helper function to translate int values
     # To enum values
     def gen_mode_state_enums(self, name_of_enum, value):
-        """Generate enum values for transmitted values"""
```

Comment:
On gthe linting job we should introduce pylint as an extra line of code coverage, to improve everyone else's work to be of the current standard.